doc/howto/.gitignore

```diff
@@ -1 +1 @@
-!*.md
+*.adoc
```

Comment:
All of the future howtos are going to be adoc files in that directory.  So we shouldn't ignore adoc files here.

Reply:
I updated the pr. now only the doc updates.